gtk-sharp-profiler-startup.patch

```diff
@@ -0,0 +1,11 @@
+diff -Naur gtk-sharp.org/gtk/gui-thread-check/profiler/gui-thread-check.c gtk-sharp/gtk/gui-thread-check/profiler/gui-thread-check.c
+--- gtk-sharp.org/gtk/gui-thread-check/profiler/gui-thread-check.c	2018-12-07 05:05:42.325500894 +0100
++++ gtk-sharp/gtk/gui-thread-check/profiler/gui-thread-check.c	2018-12-07 05:40:53.044438289 +0100
+@@ -93,6 +93,6 @@
+ 	
+ 	mono_profiler_install (NULL, NULL);
+ 	mono_profiler_install_enter_leave (simple_method_enter, NULL);
+-	mono_profiler_set_events (MONO_PROFILE_ENTER_LEAVE);
++	mono_profiler_set_events (1 << 12);
```

Comment:
If this was made private API does it even work or is it supported? Did upstream gtk-sharp patch this?

Reply:
@TingPing 

> Did upstream gtk-sharp patch this?

Unfortunately, It seems that the upstream vendor does not care too much about gtk# 3.0 branch

@Therzok [said](https://github.com/mono/gtk-sharp/issues/243#issuecomment-442269125):
> I have personally not done any maintenance work towards the gtk# 3.0 branch.

Reply:
@TingPing 

See also:
https://github.com/mono/mono/commit/ea4e4a9ef6fc42570a23026adbe826cf7248290e#diff-eda91c868134bef7be0896b88743525cR46
https://github.com/mono/mono/blob/dc2e330a9ff2d5c5271693d5b8d685aa8c0dd3b2/mono/metadata/profiler.h#L29

Arch maintainers decided to remove this line:
https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/gtk-sharp-3&id=b21d224a529f163f746d384703b8649f4ff77a3a#n18

Reply:
Patched upstream yesterday:
https://github.com/mono/gtk-sharp/pull/266